gazebo/igvc_description/urdf/Jaymii.urdf

```diff
@@ -392,63 +392,36 @@
   </transmission>
 
 
-  <joint name="base_footprint_to_base_link" type="fixed">
-    <parent link="base_footprint"/>
-    <child link="base_link"/>
-    <origin xyz="0 0 0.12" rpy="0 0 0"/>
+  <joint name="imu_to_base_footprint" type="fixed">
```

Comment:
Do you mind if we flip this so that base_footprint is the parent of the TF tree. It helps with gazebo things.